src/redux/store.ts

```diff
@@ -22,16 +17,21 @@ import { FSAction } from './fs/Actions'
 import fsReducer from './fs/FSReducer'
 import { migrateEachYear } from './migration'
 
-const migrate: PersistMigrate = async (state) =>
-  migrateEachYear(state as unknown as PersistedState & YearsTaxesState)
+export type USTPersistedState = { _persist: PersistState } & YearsTaxesState
```

Comment:
I think it's slightly more idiomatic to instead do: `NonNullable<PersistedState> & YearsTaxesState`